package.json

```diff
@@ -74,16 +77,14 @@
     "@types/react-dom": "18.3.0",
     "@types/react-virtualized-auto-sizer": "^1.0.1",
     "@types/react-window": "^1.8.5",
-    "@typescript-eslint/eslint-plugin": "7.18.0",
```

Comment:
this is a typo ? these plugins are actually crutial for us

Reply:
it was automatically removed during running nx generator for moving to flat config. The reason is that new [typescript-eslint](https://typescript-eslint.io/packages/typescript-eslint/) already includes those packages.